{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--1ac30aa7-b59c-4597-a6cb-c88db45714fd",
    "created": "2020-05-26T20:29:46.789303Z",
    "modified": "2020-05-26T20:29:46.789303Z",
    "name": "CVE-2018-14034",
    "description": "An issue was discovered in the HDF HDF5 1.8.20 library. There is an out of bounds read in the function H5O_pline_reset in H5Opline.c."
}